Node-DC-EIS-client/node_dc_eis_testurls.py

```diff
@@ -182,7 +182,7 @@ def post_function(url, post_data):
     print e
   return r
 
-def post_url(url,url_type,request_num,log,phase): 
```

Comment:
@egalli @priyankasprakash Why did we remove "log" parameter from these functions but kept as an argument in util.printlog()?

Reply:
We used to pass it (which makes a copy of the shared global). This was making it hard to close and flush to the file at a fixed interval. Now everyone uses the global directly.